node-tests/test.js

```diff
@@ -24,17 +25,16 @@ describe('ember-cli-eslint', function() {
     return emberTest().then(function(result) {
       expect(result.error).to.not.exist;
       expect(result.stdout.match(/[^\r\n]+/g))
-        .to.contain(`ok 1 ${browser} - ESLint | app.js: should pass ESLint`)
-        .to.contain(`ok 2 ${browser} - ESLint | controllers/thing.js: should pass ESLint`)
-        .to.contain(`ok 3 ${browser} - ESLint | helpers/destroy-app.js: should pass ESLint`)
-        .to.contain(`ok 4 ${browser} - ESLint | helpers/module-for-acceptance.js: should pass ESLint`)
-        .to.contain(`ok 5 ${browser} - ESLint | helpers/resolver.js: should pass ESLint`)
-        .to.contain(`ok 6 ${browser} - ESLint | helpers/start-app.js: should pass ESLint`)
-        .to.contain(`ok 7 ${browser} - ESLint | models/thing.js: should pass ESLint`)
-        .to.contain(`ok 8 ${browser} - ESLint | resolver.js: should pass ESLint`)
-        .to.contain(`ok 9 ${browser} - ESLint | router.js: should pass ESLint`)
-        .to.contain(`ok 10 ${browser} - ESLint | test-helper.js: should pass ESLint`)
-        .to.not.contain(`not ok 11 ${browser} - ESLint | unused.js: should pass ESLint`);
+        .to.include.something.that.matches(/ok 1 Chrome [0-9]+\.[0-9]+ - \[[0-9]+ ms\] - ESLint \| app.js: should pass ESLint/)
```

Comment:
I'm not sure I understand why we need `chai-things`. `result.stdout` is a multiline string, and from my understanding we could do something like:

```js
expect(result.stdout)
  .to.match(/ok 1 [^\n]+ - ESLint | app\.js: should pass ESLint/)
  .to.match(/ok 2 [^\n]+ - ESLint | foo\.js: should pass ESLint/);
```

Reply:
Hmm... we are testing the ESLint output here, correct? Seems to me like we should ensure these statements are on separate lines.

Reply:
okay
```js
expect(result.stdout)
  .to.match(/ok 1 [^\n]+ - ESLint | app\.js: should pass ESLint\r?\n/)
  .to.match(/ok 2 [^\n]+ - ESLint | foo\.js: should pass ESLint\r?\n/);
```

Reply:
will do.

Reply:
done!